src/migration-schema.ts

```
@@ -6,6 +6,7 @@ const { migrationsDir } = config
 export type MigrationSchema = {
   table: string;
   operation: string;
+  runner: string;
```

let's use `mysql` as default and hide it from user